pkg/util/util_test.go

```diff
@@ -125,23 +126,143 @@ func TestConvertTagsToMap(t *testing.T) {
 }
 
 func TestMakeDir(t *testing.T) {
-	targetTest := "./target_test"
-	//Successfully create directory
-	err := MakeDir(targetTest)
-	assert.NoError(t, err)
-
-	// Remove the directory created
-	err = os.RemoveAll(targetTest)
-	assert.NoError(t, err)
+	testCases := []struct {
+		desc          string
+		setup         func()
+		targetDir     string
+		expectedError bool
+		cleanup       func()
+	}{
+		{
+			desc:          "should create directory",
+			targetDir:     "./target_test",
+			expectedError: false,
+		},
+		{
+			desc: "should not return error if path already exists",
+			setup: func() {
+				err := os.Mkdir("./target_test", os.FileMode(0755))
+				assert.NoError(t, err, "Failed in setup: %v", err)
+			},
+			targetDir:     "./target_test",
+			expectedError: false,
+		},
+		{
+			desc: "[Error] existing file in target path",
+			setup: func() {
+				_, err := os.Create("./file_exists")
```

Comment:
You'll need to call `Close()` on the returned `File` object in order to be able to delete it on line 159 below. I believe this is the reason the Windows unit tests are failing to delete the file with an "in-use" error.

Reply:
Ah! Thank you for pointing this out. I forgot about closing the file pointer 👍